src/components/sidebar/right-sidebar/new-datepicker/NewDateSlider.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
```

Comment:
Also nit but let's just call the components `DatePicker` and `DateSlider`, they won't be new forever lmao

Reply:
yeah good point LOL, ill remove the old ones and update the names